library/include/ck/library/tensor_operation_instance/gpu/grouped_convolution_backward_weight.hpp

```diff
@@ -276,15 +276,29 @@ struct DeviceOperationInstanceFactory<ck::tensor_operation::device::DeviceGroupe
                              is_same_v<OutDataType, float> && is_same_v<ComputeTypeA, float> &&
                              is_same_v<ComputeTypeB, float>)
                 {
-                    add_device_grouped_conv1d_bwd_weight_xdl_gnwc_gkxc_gnwk_f32_instances(op_ptrs);
+                    add_device_grouped_conv1d_bwd_weight_xdl_gnwc_gkxc_gnwk_f32_default_pipev2_instances(
```

Comment:
Can we reduce number of in instances of GNHWC layout to previous number (Only for v1)?